### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by woojeonghyuk**

### CODE REVIEW

**Comment by JunheePark0224** [comment]
I think this content is not proper for contributing.md. Can you change it?

**Comment by woojeonghyuk** [comment]
Ok, I'll change it

**Comment by Inyong-Hwang** [comment]
Good

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,68 @@
-이거 내가 수정했엉 확인좀
```
Comment (JunheePark0224) [inline CONTRIBUTING.md:1]: What is this?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,68 @@
-이거 내가 수정했엉 확인좀 
+# Contributions
```
Comment (JunheePark0224) [inline CONTRIBUTING.md:1]: I think we should make better contributing.md file!

**Review by JunheePark0224 — APPROVED** [review]

**Pull request closed without merging**